h2o-automl/src/test/java/ai/h2o/automl/AutoMLTest.java

```diff
@@ -192,6 +192,58 @@
     }
   }
 
+
+  @Test public void test_individual_model_max_runtime() {
+    AutoML aml=null;
+    Frame fr=null;
+    try {
+      AutoMLBuildSpec autoMLBuildSpec = new AutoMLBuildSpec();
+//      fr = parse_test_file("./smalldata/prostate/prostate_complete.csv"); //using slightly larger dataset to make this test useful
+//      autoMLBuildSpec.input_spec.response_column = "CAPSULE";
+      fr = parse_test_file("./smalldata/diabetes/diabetes_text_train.csv"); //using slightly larger dataset to make this test useful
+      autoMLBuildSpec.input_spec.response_column = "diabetesMed";
+      autoMLBuildSpec.input_spec.training_frame = fr._key;
+
+      int max_runtime_secs_per_model = 10;
+      autoMLBuildSpec.build_models.exclude_algos = new Algo[] {Algo.DRF};
```

Comment:
@seb-h2o why are we excluding DRF here?

Reply:
another investigation leftover: removing.